### ISSUE TITLE
how we can test or predict any image?

### ISSUE THREAD

**Issue by lan786** [Horea94/Fruit-Images-Dataset#23]
Hi @Horea94  Thanks for this wonderful work. Training is ok but how I can test any image? Seems that there is no code for testing.

**Comment by Horea94** [comment]
I added a [test script](https://github.com/Horea94/Fruit-Images-Dataset/blob/master/src/image_classification/Test%20Model.py) under src/image_classification. This script is pretty straightforward, as it loads the saved model and makes a prediction on a single image.
If you want to test multiple images, I would recommend using a data generator. ([Documentation](https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/image/ImageDataGenerator))

**Comment by lan786** [comment]
@Horea94  Thanks so much for this update. I run this file and its working fine. I'm getting result in terminal mode, how to save resultant image? I mean how to get predicted image with detected box around the object?

**Comment by Horea94** [comment]
The model in this repo is a classifier. That is, it provides only a label prediction based on an input image.
For bounding box prediction, you would need a detector like [Faster R-CNN](https://arxiv.org/abs/1506.01497), [YOLO](https://arxiv.org/abs/1506.02640) or [SSD](https://arxiv.org/abs/1512.02325).
